docs/paper/dev/getting-started/manual-project-setup.md

```diff
@@ -1,9 +1,9 @@
 ---
-slug: /dev/project-setup
+slug: /dev/manual-project-setup
```

Comment:
redirect might be required here?

Reply:
from to? i cannot follow.

Reply:
yes, you cannot just change slugs. You need to set a redirect in the vercel file